src/AlertCard/AlertCard.tsx

```diff
@@ -0,0 +1,234 @@
+/** @jsx jsx */
```

Comment:
I'm commenting here just so the conversation can be threaded. I'm confused as to why this is named `AlertCard`. Which item in the design principals docs is this implementing and what naming conflict was there?

https://apollographql.quip.com/AgFcAwybcEWN/Alerts-design-principles

Reply:
These are to be used in the Alert toast cards.
Joel pointed out the conflict
> The other thing I noticed is that in SpaceKit this component is listed as an Alert, but technically it's a specific type of alert (alert toast card) out of two currently designed for types (alert banner and alert toast cards). Is your intention to rename these to Alert Toast Cards or a similar designation after we add the alert banners?

We will need an AlertBanner soon so `AlertCard` and `AlertBanner` seemed preferable to `Alert` and `AlertBanner`